src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,64 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
+    
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final int FIRST_ELEMENT = 0;
+    private static final int SECOND_ELEMENT = 1;
+    private int summaSuply = 0;
+    private int summaBuy = 0;
+    
     public void getStatistic(String fromFileName, String toFileName) {
-
+        
+        String[] readToArray = readFile(fromFileName).split("\n");
+        
+        for (String s : readToArray) {
+            String[] dropLine = s.split(" ");
+            if ((SUPPLY.equals(dropLine[FIRST_ELEMENT]))) {
+                summaSuply = summaSuply + Integer.parseInt(dropLine[SECOND_ELEMENT]);
+            } else {
+                summaBuy = summaBuy + Integer.parseInt(dropLine[SECOND_ELEMENT]);
+            }
+        }
```

Comment:
```suggestion
        String report = createReport(dataFromFile);
```
lets create a separate method to create a report, and move this logic there
ps. variables summaSupplu an summaBuy should be local there